explorer/models/taskNode.ts

```diff
@@ -0,0 +1,66 @@
+import { ResourceManagementClient, SubscriptionClient, SubscriptionModels } from 'azure-arm-resource';
+import * as opn from 'opn';
+import * as vscode from 'vscode';
+import * as ContainerModels from '../../node_modules/azure-arm-containerregistry/lib/models';
+import { AzureAccount, AzureSession } from '../../typings/azure-account.api';
+import { AzureCredentialsManager } from '../../utils/azureCredentialsManager';
+import { NodeBase } from './nodeBase';
+
+/*Single TaskRootNode under each Repository. Labeled "Build Tasks" */
+export class TaskRootNode extends NodeBase {
+    constructor(
+        public readonly label: string,
+        public readonly contextValue: string,
+        public subscription: SubscriptionModels.Subscription,
+        public readonly azureAccount: AzureAccount,
+        public registry: ContainerModels.Registry,
+        public readonly iconPath: any = {}
+    ) {
+        super(label);
+    }
+
+    public name: string;
+
+    public getTreeItem(): vscode.TreeItem {
+        return {
+            label: this.label,
+            collapsibleState: vscode.TreeItemCollapsibleState.Collapsed,
+            contextValue: this.contextValue,
+            iconPath: this.iconPath
+        }
+    }
+
+    /*Making a list view of BuildTaskNodes, or the Build Tasks of the current registry */
+    public async getChildren(element: TaskRootNode): Promise<BuildTaskNode[]> {
+        const buildTaskNodes: BuildTaskNode[] = [];
+        let buildTasks: ContainerModels.BuildTask[] = [];
+
+        const client = AzureCredentialsManager.getInstance().getContainerRegistryManagementClient(element.subscription);
+        const resourceGroup: string = element.registry.id.slice(element.registry.id.search('resourceGroups/') + 'resourceGroups/'.length, element.registry.id.search('/providers/'));
+
+        buildTasks = await client.buildTasks.list(resourceGroup, element.registry.name);
+        if (buildTasks.length === 0) {
+            vscode.window.showErrorMessage(`You do not have any Build Tasks in '${element.registry.name}'. You can create one with ACR Build. `, "Learn More").then(val => {
```

Comment:
Is showErorrMessage the only option? Does it support showInfoMessage?

Reply:
It's the only option right now. Do you want me to change it to showInfo?

Reply:
Do you mean there is a vscode.window has a method called showInfo?